hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/KMSClientProvider.java

```diff
@@ -17,7 +17,7 @@
  */
 package org.apache.hadoop.crypto.key.kms;
 
-import io.netty.util.internal.StringUtil;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
move this one line down to try and keep imports in order. its a losing battle. but we try...

Reply:
Done.